_includes/provider-ocp-add-container.md

```diff
@@ -109,7 +109,7 @@
 
 9. In the **Virtualization** endpoint tab optionally configure the virtualization service details:
 
-    1. Select a service type (KubeVirt) or leave disabled
+    1. Select a service type (OpenShift Virtualization (KubeVirt)) or leave disabled
```

Comment:
@agrare This requires a code change.

@qasim-ahmed Can we change this one to say `OpenShift Virtualization / KubeVirt`?  That's what the UI value will be when we make the code change, so I want it to match here.

Reply:
https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/532